db/schema.rb

```diff
@@ -393,8 +394,8 @@
     t.string "line_channel_id", null: false
     t.string "line_channel_secret", null: false
     t.string "line_channel_token", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
```

Comment:
lets remove the precision: nil changes in `schema.rb` and only keep the relavant ones.